packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -124,6 +135,49 @@ module.exports = ({ types: t }) => {
         ],
       },
 
+      AssignmentExpression(path) {
+
+        const right = path.get('right');
+        const left = path.get('left');
+
+        const canShorten = (
+          right.type === 'BinaryExpression' &&
+          operators.has(right.node.operator) &&
+          left.node.name === right.node.left.name
+        );
+
+		    if (!canShorten) return;
+
+        const canBeUpdateExpression = (
+          right.node.right.type === 'NumericLiteral' &&
+          right.node.right.value === 1 &&
+          updateOperators.has(right.node.operator));
+
+        if (left.node.name === undefined) {
```

Comment:
`left.isMemberExpression()` is better IMO, rather than depending on left not being an identifier.